e2e/fixtures.cy.ts

```diff
@@ -0,0 +1,40 @@
+beforeEach(() => {
+  cy.visit('/');
+  cy.wait('@fixtures');
+  cy.get('.nav-link').contains('Fixtures').click();
+});
+
+describe('fixtures', () => {
+  it('displays fixtures', () => {
+    cy.get('[data-cy="fixture-group"]')
+      .should('have.length', 6)
+      .first()
+      .within(() => {
+        cy.contains('Bundesliga').should('be.visible');
+        cy.get('[alt="League Logo"]').should('be.visible');
+        cy.get('a')
+          .should('have.attr', 'href', '/fixtures/1049002')
+          .within(() => {
+            cy.get('[alt="Home team logo"]').should('be.visible');
+            cy.get('[alt="Away team logo"]').should('be.visible');
+            cy.contains('FC Koln').should('be.visible');
+            cy.contains('FSV Mainz 05').should('be.visible');
+            cy.contains('Sun Dec 10 2023').should('be.visible');
+          });
+      });
+  });
+
+  it('shows goals for fixture', () => {
+    cy.get('[data-cy="fixture-group"]').should('have.length', 6).first().click();
+    cy.location().should((loc) => {
+      expect(loc.pathname).to.eq('/fixtures/1049002');
+    });
+
+    cy.wait(['@goals', '@goals', '@goals']).then((interceptions) => {
```

Comment:
Why does it wait for 3 goals requests? Locally with react strict mode there would be 2 and on prod there is only 1. Maybe its because it never waited for the initial one on page load? If so, should we ad a wait for that in the `beforeEach` at the top?

Reply:
Not sure what you mean about strict mode - when I run it in dev mode locally there are two requests to goals on page load, then when we click there is a third. 

These tests always run against a dev build of top90 - yeah we could put another wait in the beforeEach hook

Reply:
Actually, putting the wait in beforeEach gets kind of complicated. There's a test in the suite that needs to assert against the first request to goals 🤔

Reply:
Discussed offline but yeah strict mode is [this](https://github.com/wweitzel/top90-frontend/blob/c5c3609fb1b5eaa87bf2dabcc3e7fd70d1eb4cda/src/index.tsx#L10) and causes useEffects to be run twice in dev.

So at this point in the test there are actually a total of 4 goals requests when running in dev/strict mode:
- Test loads the goals page causing 2 @goals requests
- Test loads the fixtures page causing 2 more @goals requests

Fine for now but we should really think about a solution to this because it is confusing having to keep track of the double requests.